power-pay-backend/src/main/java/com/adorsys/gis/powerpay/powerpaybackend/services/TransactionService.java

```diff
@@ -0,0 +1,33 @@
+package com.adorsys.gis.powerpay.powerpaybackend.services;
+
+import com.adorsys.gis.powerpay.powerpaybackend.domain.Transaction;
+import com.adorsys.gis.powerpay.powerpaybackend.repository.TransactionRepository;
+
+public class TransactionService {
+    private final MoneyTransferInterface moneyTransferInterface;
+    private final TransactionRepository transactionRepository;
+
+    public TransactionService(MoneyTransferInterface moneyTransferInterface, TransactionRepository transactionRepository) {
+        this.moneyTransferInterface = moneyTransferInterface;
+        this.transactionRepository = transactionRepository;
+    }
```

Comment:
You should use lombok for this kind of stuffs

Reply:
Ok @stephane-segning noted